crypto/hd/algorithm_test.go

```diff
@@ -122,3 +125,43 @@ func TestKeyring(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, addr.String(), account.Address.String())
 }
+
+func TestHDPath(t *testing.T) {
+	params := *hd.NewFundraiserParams(0, ethermint.Bip44CoinType, 0)
+	// need to prepend "m/" because the below method provided by the sdk does not add the proper prepending
+	hdPath := "m/" + params.String()
+	require.Equal(t, "m/44'/60'/0'/0/0", hdPath)
+}
+
+func TestDerivation(t *testing.T) {
+
+	mnemonic := "picnic rent average infant boat squirrel federal assault mercy purity very motor fossil wheel verify upset box fresh horse vivid copy predict square regret"
+
+	bz, err := DeriveSecp256k1(mnemonic, keys.DefaultBIP39Passphrase, "m/44'/60'/0'/0/0")
```

Comment:
can you add a test case that compares the addresses generated with and without the `m/` prefix to be not equal?

Reply:
added the test